README.md

```diff
@@ -58,180 +29,90 @@ After you're finished please remove all the comments and instructions!
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+# 📖 [Capstone_project_1] <a name="about-project"></a>
 
-## 🛠 Built With <a name="built-with"></a>
+> It is a project build with HTML,CSS and Javascript according to linters standards. Mobile responsive website version.
 
-### Tech Stack <a name="tech-stack"></a>
+**[Capstone_project_1]** is a site practice of my own from Microverse
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
+## 🛠 Built With HTML,CSS and Javascript <a name="built-with"></a>
 
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
 
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-
-<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+- **[Build with semantic HTML tags]**
+- **[CSS responsive media queries added]**
+- **[CSS flexbox module used in this project]**
+- **[.gitignore has been set up properly]**
+- **[JS functionality added properly]**
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://google.com)
+- https://rifat8080.github.io/
```

Comment:
- [ ] Please add the full path of your live project it seems like you missed the repo name in the project live link. Kindly add the correct path to make your `documentation` looks professional.